Tools/MGCB/SourceFileCollection.cs

```diff
@@ -21,9 +21,14 @@ public sealed class SourceFileCollection
         [XmlArrayItem("File")]
         public List<string> SourceFiles { get; set; }
 
+        [XmlArrayItem("File")]
+        public List<string> DestFiles { get; set; }
```

Comment:
No, it will serialize them as follows:
```
<SourceFiles>
   <File> asdf </File>
</SourceFiles>
<DestFiles>
   <File> asdf </File>
</DestFiles>
```